<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUsuario">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Usuário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSenha">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logar</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="teHora">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="minimumTime">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="deData">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2030</year>
      <month>12</month>
      <day>30</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2023</year>
      <month>11</month>
      <day>26</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Agendamento</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCheckLavagem">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apenas Lavagem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tipo de serviço</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCheckSecagem">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apenas Secagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAgendar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agendar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatusLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
